backend-shared/config/db/db_field_constants.go

```diff
@@ -90,3 +90,90 @@ func TruncateVarchar(s string, maxLength int) string {
 
 	return s
 }
+
+// A Map object to be used by utils.ValidateFieldLength function.( Map[<Constant Variable Name as String>] <Constant Variable> )
+// At run time we need to check if number of characters in user input are within the max limit defined by constant variables above.
+// Because constant variable names (Ex: ClusterUserUserNameLength) are not same as field names (Ex: ClusterUser.User_Name) of object,
+// we need to format field name to get constant variable name we need (i.e <ObjectType><FieldName><Length>),
+// but issue here that after formating we will get constant variable name as a String, and golang does not support eval() functionality similar to Node, Python etc,
+// we need to create a Map[<Constant Variable Name as String>] <Constant Variable> object.
+var dbFieldMap = map[string]int{
```

Comment:
This is a good solution, and thanks for the detailed comment here!

I think we need a CI check to make sure that developers don't forget to add new database fields to this list. I can easily see someone adding a field to the `db-schema.sql` file, and to the constants above, but forgetting to update this map.
- Why? If we don't do this, then when we add a new field to the database, we may forget to update the map, and the length value for the field will be returned as 0 by GetConstantValue (thus breaking the functionality that uses it). 

So, can you add some quick code to to `hack/db-schema-sync-check/main.go` in backend-shared, to verify that constants defined at the top of the `db_field_constants.go` are also included in this dbFieldMap?

You should just be able to do some simple string parsing, similar to what was done of part of the `db-schema-sync-check/main.go`. Then, to execute the check, run `run-db-schema-sync-check.sh` in `hack/`.